src/cli/translate.ts

```
@@ -344,90 +476,123 @@ export const translateCommand = new Command('translate')
-            }
-
-            if (typeof newFrontmatter.title !== 'string') {
-              delete newFrontmatter.title
+            if (shouldCopyOnly) {
+              logger.info(
+                `Copying ${cyan(sourceRelativePath)} to ${cyan(targetRelativePath)}`,
+              )
+
+              // For copy-only files, we still update the sourceSHA but don't translate
+              const newFrontmatter = { ...sourceFrontmatter, sourceSHA }
+              delete newFrontmatter.i18n
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)